napari/_qt/qt_viewer.py

```diff
@@ -358,7 +359,9 @@ def _create_canvas(self) -> None:
         self.canvas.events.mouse_double_click.connect(
             self.on_mouse_double_click
         )
-        self.canvas.events.mouse_move.connect(self.on_mouse_move)
+        self.canvas.events.mouse_move.connect(
+            qthrottled(self.on_mouse_move, timeout=20)
+        )
```

Comment:
Did you test if it will work correctly? If fast mouse move will not create holes? Maybe there should be some additional logic that will fire the event if the distance between two points is enough big?